mw-webapp/src/component/header/Header.tsx

```diff
@@ -195,25 +195,24 @@ export const Header = (props: HeaderProps) => {
           name={LOGO_TEXT}
         />
       </Link>
-      <div className={styles.headerButtonsContainer}>
-
-        <ThemeSwitcher
-          language={props.language}
-          theme={props.theme}
-          setTheme={props.setTheme}
-          className={styles.themeSwitcher}
-          dataCy={headerAccessIds.settings.themeSwitcher}
-        />
-
-        <Select
-          value={props.language}
-          name="language"
-          options={languageOptions}
-          onChange={props.setLanguage}
-          className={styles.selectLanguage}
-        />
-
-        <HorizontalContainer className={styles.rightBlock}>
+      <HorizontalContainer className={styles.headerButtonsContainer}>
+        <HorizontalContainer className={styles.headerLeftBlock}>
```

Comment:
let's rename for more semantic (without lefy right, etc) Because if we will replace these element then classes will b e wrong

Reply:
fixed